packages/strapi-design-system/src/Field/FieldLabel.js

```diff
@@ -17,19 +17,18 @@ const Action = styled(Flex)`
   }
 `;
 
-export const FieldLabel = ({ children, required, action, ...props }) => {
-  const { id } = useField();
+export const FieldLabel = ({ children, action, required: requiredDeprecatedProp, ...props }) => {
+  const { id, required: requiredField } = useField();
+  const required = requiredField || requiredDeprecatedProp;
+
+  if (requiredDeprecatedProp !== undefined) {
```

Comment:
I added a console.warn here to inform about the deprecation, let me know what you think @gu-stav